include/mbedtls/x509_crt.h

```diff
@@ -331,12 +331,22 @@ typedef void mbedtls_x509_crt_restart_ctx;
 /**
  * Default security profile. Should provide a good balance between security
  * and compatibility with current deployments.
+ *
+ * This profile permits:
+ * - SHA2 hashes with at least 256 bits: SHA-256, SHA-384, SHA-512.
+ * - Elliptic curves with 255 bits and above.
+ * - RSA with 2048 bits and above.
```

Comment:
I'm not sure if we should up this to 3072.

Reply:
Well according to [SSL pulse](https://www.ssllabs.com/ssl-pulse/) ("Key strength distribution"), 2048 is still the most frequent size so I don't think we can require more than that.